Tools-Override/tests.targets

```diff
@@ -24,11 +24,16 @@
     <DefaultNoCategories Include="$(DefaultNoCategories)" />
     <UnsupportedPlatformsItems Include="$(UnsupportedPlatforms)"/>
   </ItemGroup>
+  
+  <ItemGroup>
+    <SupplementalTestData Include="$(RuntimePath)\xunit.console.netcore.exe" />
```

Comment:
Could you explain this part a little more? It seems like this is treated a bit specially from the other stuff.

Unrelatedly, I think @weshaggard was hoping we could just start using `Content` for everything, and phase out the usage of `SupplementalTestData`.